### ISSUE TITLE
Key System is half broken..

### ISSUE THREAD

**Issue by xRyukan000** [SPDM-Team/Arceus-X-NEO-public#1]
So to get a key it takes you to linkvertise but linkvertise only lets you use it free 3 times a day. Its not free so you cant get keys. I would request changing what website you have the key system in please.

**Comment by TeensConnectWeb** [comment]
you can choose to use lootlink or linkvertise each step

**Comment by xRyukan000** [comment]
Yeah that's what I've been doing but its just that I emailed every executor
about this "glitch"? when it happened

**Comment by synergyatomic** [comment]
Please make an original key system Linkvertise Now it only gives us 2 credits so please make a lootlinks key system, Readme
